cdap-ui/app/directives/my-pipeline-configurations/my-realtime-pipeline-config/my-realtime-pipeline-config-ctrl.js

```diff
@@ -205,24 +205,24 @@ class MyRealtimePipelineConfigCtrl {
     this.updatePipelineEditStatus();
   }
 
-  onCheckPointingChange = () => {
+  onCheckPointingChange() {
     this.checkpointing = !this.checkpointing;
     this.updatePipelineEditStatus();
-  };
+  }
 
-  onInstrumentationChange = () => {
+  onInstrumentationChange() {
     this.instrumentation = !this.instrumentation;
     this.updatePipelineEditStatus();
-  };
+  }
 
-  onStageLoggingChange = () => {
+  onStageLoggingChange() {
     this.stageLogging = !this.stageLogging;
     this.updatePipelineEditStatus();
-  };
+  }
 
-  onBackpressureChange = () => {
+  onBackpressureChange() {
```

Comment:
because of this revert, do you have to add `.bind` to the constructor for these methods?

Reply:
For some reason was testing my UI changes on the sdk node process.